client/lib/browser-storage/index.ts

```diff
@@ -10,6 +10,8 @@ import {
 	activate as activateBypass,
 } from './bypass';
 import { StoredItems } from './types';
+import { mc } from 'lib/analytics';
```

Comment:
🤔 Out of curiosity what happens when we comment out the import in http://iscalypsofastyet.com/branch?branch=add/metrics-on-idb-failure. The bundle size increase is rather suspicious but I wonder if lib/analytics is actually a weird size.

Maybe also rebase with master to see if the bot is happier.